README.md

```diff
@@ -170,16 +200,17 @@ To create and play with a Test object from the example package,
 To pass extra parameters to the plugin, use a comma-separated
 parameter list separated from the output directory by a colon:
 
-
 	protoc --go_out=plugins=grpc,import_path=mypackage:. *.proto
 
-
 - `import_prefix=xxx` - a prefix that is added onto the beginning of
-  all imports. Useful for things like generating protos in a
-  subdirectory, or regenerating vendored protobufs in-place.
+  all imports. Supported for backwards compatibility, but rarely useful.
```

Comment:
Perhaps use the word "deprecated" somewhere in here?

Reply:
Moved to a "deprecated" section.